examples/agents/inflation.py

```diff
@@ -23,12 +23,15 @@ async def run_main(host: str, port: int, disable_safety: bool = False):
 
     available_shields = [shield.identifier for shield in client.shields.list()]
     if not available_shields:
-        print(f"No available shields. Disable safety.")
+        print("No available shields. Disable safety.")
     else:
         print(f"Available shields found: {available_shields}")
+    available_models = [model.identifier for model in client.models.list()]
+    selected_model = available_models[0]
```

Comment:
same thing, this likely needs to be another common file? but we hate common files in the examples :/

Reply:
ugh yeah, I really hate the individual scripts here, too many updates whenever SDK changes. maybe need to move to a common llama-stack-client helper as follow up